tests/serving/test_remote.py

```diff
@@ -150,3 +154,167 @@ def test_remote_advance(httpserver, engine):
     server.wait_for_completion()
     print(resp)
     assert resp == {"req": {"url": "/dog", "data": {"x": 5}}, "resp": {"post": "ok"}}
+
+
+def _sleeping(request: Request):
```

Comment:
```suggestion
def _timed_out_handler(request: Request):
```
or `_sleeping_handler` or `_slow_handler` or something

Reply:
done